src/main/kotlin/frc/robot/subsystems/shooter/turret/Turret.kt

```diff
@@ -0,0 +1,31 @@
+package frc.robot.subsystems.shooter.turret
+
+import com.ctre.phoenix6.controls.MotionMagicTorqueCurrentFOC
+import edu.wpi.first.units.Units
+import edu.wpi.first.units.measure.Angle
+import edu.wpi.first.wpilibj.DigitalInput
+import edu.wpi.first.wpilibj2.command.SubsystemBase
+import edu.wpi.first.wpilibj2.command.button.Trigger
+import frc.robot.lib.universal_motor.UniversalTalonFX
+import org.littletonrobotics.junction.Logger
+
+class Turret : SubsystemBase() {
+    val motor = UniversalTalonFX(MOTOR_PORT, config = MOTOR_CONFIG)
+    var angleSetpoint = Units.Rotations.zero()
+    val sensor = DigitalInput(SENSOR_PORT)
+    val motionMagicTorque = MotionMagicTorqueCurrentFOC(0.0)
+    val isAtResetPoint =
+        Trigger { sensor.get() }
+            .onTrue(runOnce { motor.reset(Units.Rotations.zero()) })
```

Comment:
```suggestion
            .onTrue(runOnce { motor.reset(0.rot) })
```
It doesn't matter here but I'd use degrees

Reply:
fb292d5332744cbe8a96f9a98341e7c5514c67ba